package.json

```diff
@@ -176,16 +177,18 @@
     "defaults"
   ],
   "scripts": {
-    "build": "concurrently npm:gen:op npm:gen:shadowdom",
+    "build": "concurrently npm:gen:op npm:gen:shadowdom npm:gen:ts",
     "test": "ava test/basic.test.cjs",
     "bundle": "concurrently npm:lib:* -m 25 && concurrently npm:shadow:* -m 25 && npm run module:js",
     "bundle:pre-edit": "json -I -f package.json -e \"this.unpkg=''\"",
     "bundle:post-edit": "json -I -f package.json -e \"this.unpkg='open-props.min.css'\"",
-    "gen:op": "cd build && node props.js \"\" true",
-    "gen:nowhere": "cd build && node props \"\" false",
-    "gen:shadowdom": "cd build && node props \"\" false \":host\" \"shadow\"",
-    "gen:prefixed": "cd build && node props.js \"op\" true",
-    "module:js": "npm run bundle:pre-edit && microbundle --no-sourcemap && npm run bundle:post-edit",
+    "bundle:fix-dts-name": "cd dist && node -e \"require('fs').renameSync('index.d.ts', 'open-props.d.ts')\"",
```

Comment:
this part is a bit weird. microbundle names the types files `index.d.ts` even though the `"types"` field above says `open-props.d.ts`. so i'm just renaming it 😄